ZenPacks/zenoss/LinuxMonitor/tests/plugindata/linux/build.zenoss.loc/ifconfig

```diff
@@ -1,4 +1,4 @@
-/sbin/ifconfig -a && echo __COMMAND__ && /bin/dmesg
+export PATH=$PATH:/sbin:/usr/sbin; if which ifconfig >/dev/null 2>&1; then ifconfig -a; elif which ip >/dev/null 2>&1; then echo "### ip addr output"; ip addr; else echo "No ifconfig or ip utilities were found."; exit 127; fi && echo __COMMAND__ && /bin/dmesg
```

Comment:
The `which` command isn't a portable way to check for a command's existence. See how many results you get from searching Google for "which: command not found".

This stackoverflow article has some better suggestions. It boils down to us not even being able to assume a bash shell, so you should be using "command -v" for this purpose.

http://stackoverflow.com/questions/592620/check-if-a-program-exists-from-a-bash-script